File: data/code_pairs/cpp/2367.cpp
#include <iostream>
#include <array>
#include <string>

int main() {
    std::array<std::string, 3> a = { "Takata ", "Tanaka ", "Hayashi" };
    std::array<std::string, 3> b = { "abc", "abc", "abc" };
    std::string c = "abcd";

    // Print the values to verify
    std::cout << "Array a:" << std::endl;
    for (const auto& str : a) {
        std::cout << str << std::endl;
    }

    std::cout << "Array b:" << std::endl;
    for (const auto& str : b) {
        std::cout << str << std::endl;
    }

    std::cout << "String c:" << std::endl;
    std::cout << c << std::endl;

    return 0;
}

File: data/code_pairs/fortran/2367.f90
program test
  implicit none
  character(10) :: a(3)
  character(10) :: b(3) = (/ 'Takata ', 'Tanaka ', 'Hayashi' /)
  character(4) :: c = "abcd"

  a = (/ 'Takata ', 'Tanaka ', 'Hayashi' /)
  b = (/ 'abc', 'abc', 'abc' /)
  c = "abcd"

  print *, "Array a:"
  print *, a
  print *, "Array b:"
  print *, b
  print *, "String c:"
  print *, c

end program test